### PR TITLE
Bug/ Resolve console warnings related to Hud.js

### PR DESCRIPTION
**Pull request by rhiannonstanford**
This PR resolves warnings in console related to Hud.js
- Fix: add unique key for children of .map function 

Screen Shot 2021-11-08 at 9 04 24 PM

### CODE REVIEW

**Review by rhiannonstanford — COMMENTED** [review]
Self review.

**Review by rhiannonstanford — COMMENTED** [review]
Self review

**Review by ethankarp — APPROVED** [review]
Nice job adding the key attributes

File: src/components/pages/HeadsUpDisplay/Hud.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ function Hud(props) {
         {activities.map(a => {
           return (
             <div
+              key={`${a}`}
```
Comment (Julianisrel) [inline src/components/pages/HeadsUpDisplay/Hud.js:40]: Nice work adding the key attributes.

**Review by Julianisrel — APPROVED** [review]

**Review by milklot — APPROVED** [review]
Nice catch on div keys, thank you for fix

**Pull request merged**